src/protocol/handshake.rs

```diff
@@ -29,12 +37,8 @@ struct SecWebSocket {
 impl SecWebSocket {
     /// Generate key-accept pair for a WebSocket handshake.
     fn generate() -> Self {
-        // TODO(Rqnsom): Randomize 16 bytes.
-        let key: Vec<u8> = vec![1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16];
-        let key = base64::encode(key);
-
+        let key = tungstenite::handshake::client::generate_key();
```

Comment:
Only after I implemented my logic with the `rand` crate, I thought maybe this crate could provide this. And it does! 😀